vertex_ai/tensorboard/main.tf

```diff
@@ -16,7 +16,7 @@
 
 
 # [START aiplatform_create_tensorboard_sample]
-resource "google_vertex_ai_tensorboard" "default" {
+resource "google_vertex_ai_tensorboard" "tensorboard" {
```

Comment:
@lakeyk Thank you for the PR. Using `default` for local resources names is the standard for this repository.

You can use to `display_name` or `description` or code comments, to add more details for users.

Reply:
Ok sounds good - thanks!